.github/workflows/test-suite.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         if: steps.expo-caches.outputs.yarn-workspace-hit != 'true'
         run: yarn install --frozen-lockfile
       - name: 🔧 Install Expo CLI
-        run: yarn global add expo-cli
+        run: yarn global add expo-cli@6.1.0
```

Comment:
Web tests were breaking on new versions of expo-cli because `.mjs` files were not being resolved, probably something related to the Webpack 5 upgrade, downgrading to `6.1.0` fixes it